plugins/inputs/system/README.md

```diff
@@ -3,6 +3,10 @@
 The system plugin gathers general stats on system load, uptime,
 and number of users logged in. It is similar to the unix `uptime` command.
 
+**Note**: If using the system plugin inside a docker container, you must
```

Comment:
Let's document this at https://hub.docker.com/_/telegraf/ instead, I'm not sure that it needs to be this complicated either:

https://github.com/influxdata/telegraf/blob/master/docs/FAQ.md#q-how-can-i-monitor-the-docker-engine-host-from-within-a-container

I think if you show an example it will be easier to follow.